SetReplace/WolframModelPlot.m

```diff
@@ -15,21 +15,24 @@
 SyntaxInformation[WolframModelPlot] = {"ArgumentsPattern" -> {_, _., OptionsPattern[]}};
 
 $plotStyleAutomatic = <|
-	_ -> Hue[0.6, 0.2, 0.8], (* vertex style *)
-	_List -> Hue[0.6, 0.7, 0.5]|>; (* edge style *)
+	$vertexPoint -> Directive[Hue[0.63, 0.26, 0.89], EdgeForm[Directive[Hue[0.63, 0.7, 0.33], Opacity[0.95]]]],
+	$edgeLine -> Directive[Hue[0.63, 0.7, 0.5], Opacity[0.7]],
+	$edgePoint -> Directive[Hue[0.63, 0.7, 0.5], Opacity[0.7]],
+	$edgePolygon -> Directive[Hue[0.63, 0.66, 0.81], Opacity[0.1], EdgeForm[None]]
+|>;
 
 (* Automatic style pickes up, and possibly modifies the style it inherits from. *)
 Options[WolframModelPlot] = Join[{
 	"EdgePolygonStyle" -> Automatic, (* inherits from EdgeStyle, with specified small opacity *)
 	EdgeStyle -> Automatic, (* inherits from PlotStyle *)
 	GraphHighlight -> {},
-	GraphHighlightStyle -> Hue[1.0, 1.0, 0.7],
+	GraphHighlightStyle -> Red,
```

Comment:
Is this for event highlighting as given in the section "Event Highlight" in the notebook I sent?

Reply:
Not quite, it's a generic highlight (HypergraphPlot with Highlight section). I don't have any place to put event highlight yet, because I don't have multi-hypergraph plotting implemented yet.

But of course, I will use the correct style right away once I implement it.

Reply:
Should generic highlight be the same as event highlight?

Reply:
Okay, In the graph that Jeremy has under "HypergraphPlot with Highlight" section, he doesn't change the vertex color to Red. And something to think about - the filling in the Event highlighting doesn't change to red, is it correct to change the filling color to red in this case?

Reply:
Yes, that's correct. Vertices are different from edges in terms of how they are created/destroyed. While we can define vertices being created/destroyed in terms of events in which they first/last appear, it's not as direct as it is for edges, which is why `highlightUpdates` intentionally does not highlight vertices.

The filling should change, it's a bug. The reason is `hpBlue` defines the polygons style, which overrides styles inferred from `EdgeStyle` which are used in `highlightUpdates`. It can be fixed by setting `"EdgePolygonStyle" -> Automatic` in the `HypergraphPlot` call in `highlightUpdates`.